include/RAJA/policy/openmp/policy.hpp

```diff
@@ -192,6 +199,7 @@ using policy::omp::omp_reduce_ordered;
 
 #if defined(RAJA_ENABLE_TARGET_OPENMP)
 using policy::omp::omp_target_parallel_for_exec;
+using policy::omp::omp_target_parallel_for_exec_nt;
```

Comment:
I'm assuming `_nt` is num_teams?

Reply:
Matt experimented here by producing a policy where you "don't specify NUM_TEAMS" explicitly, and he named it thus ( no teams). But, performance didn't necessarily improve. Arguably the compiler/runtime should have a better notion of setting num teams based on grid blocks, or occupancy then just arbitrary numbers that the user selects. Setting thread count explicitly (i.e. blocksize) makes a little more sense.